fixtures/contacts.yaml

```diff
@@ -0,0 +1,9 @@
+contacts:
+    -
+        name: 311
+        email: help@austintexas.gov
+        phone:
+            default: 311
```

Comment:
data type change will require a Janis change in src/js/modules/ContactDetails and src/js/modules/ContactPhone